lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -645,9 +649,12 @@ txConstraints protocolParams witnessTag = TxConstraints
     txOutputSize bundle =
         marginalSizeOf empty {txOutputs = [mkTxOut bundle]}
 
-    txOutputMaximumSize = (<>)
-        (TxSize $ fromIntegral maxTokenBundleSerializedLengthBytes)
-        (txOutputSize mempty)
+    txOutputMaximumSize = (txOutputSize mempty <>)
+        . TxSize
+        . fromIntegral
+        . getQuantity
+        . unTokenBundleMaxSize
+        $ view (#txParameters . #getTokenBundleMaxSize) protocolParams
```

Comment:
If `TokenBundleMaximumSize` were defined in terms of `TxSize`, then we could write this more simply:

```suggestion
    txOutputMaximumSize = (<>)
        (view (#txParameters . #getTokenBundleMaxSize) protocolParams)
        (txOutputSize mempty)
```